src/mintpy/utils/utils0.py

```diff
@@ -282,9 +282,45 @@ def utm_zone2epsg_code(utm_zone):
         'south': utm_zone[-1].upper() == 'S',
     })
     epsg_code = crs.to_authority()[1]
+
+    # Link: https://gis.stackexchange.com/questions/365584/convert-utm-zone-into-epsg-code
+    # zone = '36'
+    # south = True
+    # epsg_code = 32600
+    # epsg_code += int(zone)
+    # if south is True:
+    #     epsg_code += 100
+
     return epsg_code
 
 
+def epsg_code2utm_zone(epsg_code):
+    """Convert EPSG code (in UTM coordinates) to UTM Zone string.
+
+    Parameters: epsg_code - str / int, EPSG code
+    Returns:    utm_zone  - str, atr['UTM_ZONE'], comprises a zone number
+                            and a hemisphere, e.g. 11N, 36S, etc. None for
+                            a EPSG code not in a UTM coordnate system
+    Examples:   utm_zone = epsg_code2utm_zone('32736')
+    """
+    from pyproj import CRS
+    crs = CRS.from_epsg(epsg_code)
```

Comment:
If you wanted, you could even make this `CRS.from_user_input` to accept `"EPSG:32617"` or `32617` or `"32617"` 

```

In [15]: CRS.from_epsg(32617) == CRS.from_user_input(32617) == CRS.from_user_input("32617") == CRS.from_user_input("epsg:32617")
Out[15]: True
```
and it doesn't seem any slower


```
In [8]: %timeit CRS.from_epsg(32617).utm_zone
472 µs ± 3.58 µs per loop (mean ± std. dev. of 7 runs, 1,000 loops each)

In [9]: %timeit CRS.from_user_input('EPSG:32617').utm_zone
475 µs ± 4.06 µs per loop (mean ± std. dev. of 7 runs, 1,000 loops each)

In [10]: %timeit CRS.from_user_input('32617').utm_zone
478 µs ± 9.87 µs per loop (mean ± std. dev. of 7 runs, 1,000 loops each)
```

Reply:
That's good to know. I added them in the comment. Thanks.